Makefile.toml

```diff
@@ -329,7 +340,7 @@ echo + RUSTFLAGS="${RUSTFLAGS:-<not set>}"
 set -xe
 cargo build -p ${RISEDEV_CARGO_BUILD_CRATE} -p risedev ${BUILD_HDFS_BACKEND_CMD}\
             --profile "${RISINGWAVE_BUILD_PROFILE}" \
-            --features "${RISINGWAVE_FEATURES}" \
+            ${RISINGWAVE_FEATURE_FLAGS} \
```

Comment:
I considered make `rw-static-link` default feature, and add `--no-default-features` for dynamic linking. But not sure.

Reply:
Both LGTM as long as we handle and hide all this stuff in RiseDev. 🤣